.github/workflows/release-drafter.yaml

```diff
@@ -5,8 +5,17 @@ on:
     branches:
       - dev
 
+permissions:
```

Comment:
I don't believe the `permissions` changes are needed

Reply:
No they are not needed, as that is the default for our actions.

Reply:
Revert?